src/config/theme.tsx

```diff
@@ -75,4 +75,24 @@ const theme = createMuiTheme({
   },
 });
 
+theme.typography.h6 = {
+  fontSize: '1.25rem',
+  '@media (max-width:768px)': {
+    fontSize: '1rem',
+  },
+  [theme.breakpoints.down('sm')]: {
+    fontSize: '1rem',
```

Comment:
Any reason why we are using both @media and theme.breakpoints. Do we need them both?